cmd/environments/get.go

```diff
@@ -68,6 +69,14 @@ func runGet(
 			return err
 		}
 
+		output, err := output.CmdOutput(
```

Comment:
This is the implementation. Another PR will add it for all endpoints and add functionality to work for a response with multiple resources.